### PR TITLE
Mavimaec/om v4.5.0

### PR DESCRIPTION
**Pull request by mavimaec**
# Overview

- Added new code-generated entities

### CODE REVIEW

File: src/objectModel.ts [hunk 1]
```diff
@@ -1,8 +1,4 @@
 // AUTO-GENERATED FILE, DO NOT MODIFY.
-/**
```
Comment (vadym-vorotilin) [inline src/objectModel.ts:2]: @mavimaec Could you revert this change please? I think, I added it on the phase of the npm package preparation but didn't update the code generation part.

File: src/objectModel.ts [hunk 2]
```diff
@@ -7706,7 +7702,7 @@ export interface IPhaseFilterTable {
 
 export class PhaseFilter implements IPhaseFilter {
     index: number
-    new?: number
+    _new?: number
```
Comment (vadym-vorotilin) [inline src/objectModel.ts:7708]: This case should probably be handled during the code generation phase. Otherwise we'd have to fix it manually every time.

**Review by vadym-vorotilin — CHANGES_REQUESTED** [review]

File: src/objectModel.ts (hunk 1 above)
Comment (mavimaec) [reply src/objectModel.ts:2]: Will do!

File: src/objectModel.ts (hunk 2 above)
Comment (mavimaec) [reply src/objectModel.ts:7708]: Yes, that's where I actually made the fix.

**Comment by mavimaec** [comment]
@Vadim-Vorotilin I've restored the @module vim-ts comment you mentioned. Let me know if it's all good now

**Review by vadym-vorotilin — APPROVED** [review]
Looking good!
